examples/ipython/tutorial_algebra.ipynb

```diff
@@ -0,0 +1,672 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to using $\\mathcal{G\\!A}$lgebra\n",
+    "\n",
+    "This is a tutorial to introduce you to `galgebra`, a symbolic geometric algebra library for python.\n",
+    "\n",
+    "A geometric algebra is defined by a set of symbols that represent the basis vectors of a real vector space, a metric tensor, and possible a set of coordinate symbols.  If coordinates are defined the metric tensor can be a function of them.\n",
+    "\n",
+    "The following cell imports all the functions needed in the tutorial from `sympy`, `ga`, `mv`, and `printer`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sympy\n",
+    "from galgebra.ga import Ga\n",
+    "from galgebra.printer import latex\n",
+    "from IPython.display import Math\n",
+    "\n",
+    "# tell sympy to use our printing by default\n",
+    "sympy.init_printing(latex_printer=latex, use_latex='mathjax')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Printing in sympy\n",
+    "\n",
+    "Sympy will show pretty $\\LaTeX$ renderings of symbolic expressions by default"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "$\\displaystyle n^{2}$"
+      ],
+      "text/plain": [
+       " 2\n",
+       "n "
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "sympy.S('n')**2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "But if we want to append freeform text on the same line, we must use `Math`, `latex`, and f-strings in tandem:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "$\\displaystyle y = n^{2}$"
+      ],
+      "text/plain": [
+       "<IPython.core.display.Math object>"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Math(f\"y = { latex(sympy.S('n')**2) }\")"
```

Comment:
Jupyter is not doing a good job highlighting f-string.

Reply:
Which is weird, because it looks great on my local machine. Maybe we need to update a dependency.

Reply:
![image](https://user-images.githubusercontent.com/64258/82640560-68002700-9c3d-11ea-802b-dd66c7fcd215.png)

Looks like a plain string.

Reply:
`pip install --upgrade jupyter-client pygments` maybe?

Reply:
(are you testing on your local machine, or is that a docs screenshot?)

Reply:
screenshot from https://galgebra--375.org.readthedocs.build/en/375/tutorials/algebra.html only , haven't tested it locally.

Reply:
I'll try getting readthedocs to use newer versions when I get back to my desk.

Reply:
Fixed by updating `pygments`:

![image](https://user-images.githubusercontent.com/425260/82824612-8a927a00-9ea1-11ea-9053-a26e7d67b715.png)